server/src/settings.example.cfg

```diff
@@ -1,22 +1,23 @@
 [Server]
-DEBUG = True
 HOST = 0.0.0.0
-PORT = 5000
+PORT = __$PORT__
+DEBUG = True
+WORKERS = __$WORKERS__
 
 [Version]
-VER_MAJOR = 0
-VER_MINOR = 1
-VER_PATCH = 0
+MAJOR = 0
+MINOR = 1
+PATCH = 0
 
 [Database]
-DB_CONNECTION_STRING = postgresql://REDACTED:REDACTED@localhost:5432/postgres
-DATA_DIRECTORY = static
+URL = __$DATABASE_URL__
 
-[Api]
-REDACTED = REDACTED
+[Redis]
+URL = __$REDIS_URL__
+TTL_SECONDS = 3600
 
 [Socrata]
-TOKEN = None
```

Comment:
In this scenario, what happens if i dont have settings.cfg?
It will be created and the value for TOKEN will be? empty string?

Reply:
Hey Russell, good question. The TOKEN will have whatever value SOCRATA_TOKEN has in the environment. If we're in development, that would be "REDACTED", because that's the value of SOCRATA_TOKEN in the docker-compose file. On the other hand if we're in production, it will have whatever value we set in heroku.

If SOCRATA_TOKEN is not defined in the environment, then the value of TOKEN would default to None. But there would also be a warning in the console saying that SOCRATA_TOKEN is not defined in the environment. 

The basic idea was that it's helpful to be explicit when we expect one of the config values to come from the environment, rather than having some of them be overridden implicitly. So then we always know what needs to be in the environment, which makes it easier to set up heroku and not lose track of where things are coming from.